README.md

```diff
@@ -53,7 +54,8 @@ python src/run.py
 **Body:**
 ```json
 {
-  "token": "<TOKEN received from Google>"
+  "token": "<TOKEN received from Google>",
+  "is_ios": true
```

Comment:
will there be client ID's for web or na?

Reply:
good question. not sure as of now